pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[tool.poetry]
+name = "dump-env"
+version = "0.2.2"
+description = "A utility tool to create .env files"
+
+license = "MIT"
+
+authors = [
+  "Nikita Sobolev <mail@sobolevn.me>"
+]
+
+readme = "README.md"  # Markdown files are supported
+
+repository = "https://github.com/sobolevn/dump-env"
+homepage = "https://github.com/sobolevn/dump-env"
+
+keywords = [
+  "dump-env",
```

Comment:
Keywords do not match the correct ones from https://github.com/sobolevn/dump-env/blob/master/setup.py